stdlib/2/whichdb.pyi

```diff
@@ -0,0 +1,8 @@
+# Source: https://hg.python.org/cpython/file/2.7/Lib/whichdb.py
+
+from types import ModuleType
+from typing import Optional, Text
+
+dbm = ...  # type: Optional[ModuleType]
```

Comment:
Should this be here at all? It seems to be just an imported module with fallback to None.